Fix broken build due to `config.go`

### Checklist
- [ ] Bug from Service (Vatz)
- [x] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [ ] etc(e.g, Hardware version doesn't match, )

---

### **Describe the bug**
> A clear and concise description of what the bug is.

Due to #280. Build broken.
My bad, I should check again for review.

### **How to Reproduce**
> Steps to reproduce the behavior:

```
rootwarp@mindhack ~/g/vatz main > make coverage
fatal: No names found, cannot describe anything.
echo "Test Coverage script will be here"
Test Coverage script will be here
# github.com/dsrvlabs/vatz/manager/config
manager/config/config.go:198:5: p.dupplicatedPlugin undefined (type parser has no field or method dupplicatedPlugin)
FAIL    github.com/dsrvlabs/vatz/cmd [build failed]
ok      github.com/dsrvlabs/vatz/manager/api    0.007s  coverage: 0.0% of statements [no tests to run]
FAIL    github.com/dsrvlabs/vatz/manager/config [build failed]
FAIL    github.com/dsrvlabs/vatz/manager/dispatcher [build failed]
FAIL    github.com/dsrvlabs/vatz/manager/executor [build failed]
FAIL    github.com/dsrvlabs/vatz/manager/healthcheck [build failed]
FAIL    github.com/dsrvlabs/vatz/rpc [build failed]
ok      github.com/dsrvlabs/vatz/utils  0.004s  coverage: 50.0% of statements
make: *** [Makefile:16: coverage] Error 2
```

### **Expected behavior**
> A clear and concise description of what you expected to happen.

Should be success.

### **Screenshots**
> If applicable, add screenshots to help explain your problem.

--- a/manager/config/config.go
+++ b/manager/config/config.go
@@ -195,7 +195,7 @@ func InitConfig(configFile string) (*Config, error) {
 			return
 		}
 
-		p.dupplicatedPlugin(vatzConfig)
+		p.duplicatedPlugin(vatzConfig)
 	})
 
 	wg.Wait()

